.github/ISSUE_TEMPLATE/dependency.yml

```diff
@@ -0,0 +1,103 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# See https://gh-community.github.io/issue-template-feedback/structured/
+
+name: Dependency
+title: ":arrow_up: Upgrade <artifact> from <current version> to <target version>"
+description: Keep upstream dependencies fresh and stable
+labels: [ "kind:build, priority:major, good first issue, help wanted" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for finding the time to report the issue! We really appreciate the community's efforts to improve Kyuubi.
+
+        It doesn't really matter whether what you are reporting is a bug or not, just feel free to share the problem you have
+        encountered with the community. For best practices, if it is indeed a bug, please try your best to provide the reproducible
+        steps. If you want to ask questions or share ideas, please [subscribe to our mailing list](mailto:dev-subscribe@kyuubi.apache.org)
+        and send emails to [our mailing list](mailto:dev@kyuubi.apache.org), you can also head to our
+        [Discussions](https://github.com/apache/kyuubi/discussions) tab.
+
+  - type: checkboxes
+    attributes:
+      label: Code of Conduct
+      description: The Code of Conduct helps create a safe space for everyone. We require that everyone agrees to it.
+      options:
+        - label: >
+            I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      options:
+        - label: >
+            I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar
+            issues.
+          required: true
+
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Why do we need to upgrade this artifact?
+      options:
+        - Common Vulnerabilities and Exposures (CVE)
+        - Bugfixes
+        - Usage of New Features
+        - Performance Improvements
+        - Regular Updates
+    validations:
+      required: true
+
+  - type: input
+    id: artifact
+    attributes:
+      label: Artifact Name
+      description: Which artifact shall be upgraded?
+      placeholder: e.g. spark-sql
+      value: https://mvnrepository.com/search?q=
+    validations:
+      required: true
+
+  - type: input
+    id: versions
+    attributes:
+      label: Target Version
+      description: Which version shall be upgraded?
+      placeholder: e.g. 1.2.1
+    validations:
+      required: true
+
+  - type: textarea
+    id: release_notes
+    attributes:
+      label: Release Notes
```

Comment:
this part may not limit to release note, how about "Notable Changes"?

Reply:
Sound reasonable, addressed